tests/integration/models/reddit/test_subreddit.py

```diff
@@ -232,10 +232,25 @@ def test_remove__non_special(self, _):
 
 
 class TestSubredditFlair(IntegrationTest):
+    REDDITOR = pytest.placeholders.username
+
     @property
     def subreddit(self):
         return self.reddit.subreddit(pytest.placeholders.test_subreddit)
 
+    def test__call(self):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette('TestSubredditFlair.test__call'):
+            mapping = self.subreddit.flair()
+            assert len(list(mapping)) > 0
+
+    def test_banned__callable__user_filter(self):
```

Comment:
This PR looks amazing. The only very minor thing I see is that this method isn't in sorted order relative to the other methods in `TestSubredditFlair`. Great work!